fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -139,6 +145,36 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
       getLocationSuggest,
     ]);
 
+    const loadInitialLocation = useCallback(async () => {
+      if (!initialValue || initialLocation?.id.value === initialValue) {
+        return;
+      }
+
+      if (!client) {
+        return;
+      }
+      const { data } = await client.query({
```

Comment:
I think specifying the types here is good for safety:

```suggestion
      const { data } = await client.query<
        LocationQuery,
        LocationQueryVariables
      >(
```